api/__tests__/services/account.test.ts

```
@@ -3,9 +3,21 @@ import { isValidEmail } from '../../utils/validation';
-// Mock the validation utility
+// Setup proper TypeScript Jest mocks
 jest.mock('../../utils/validation');
+jest.mock('../../services/email');
+
+// Create mock functions
+const mockedIsValidEmail = jest.fn();
+const mockedGenerateVerificationCode = jest.fn();
+const mockedSendEmail = jest.fn();
+
+// Assign mock implementations
+(isValidEmail as jest.Mock) = mockedIsValidEmail;
```

Biome (1.9.4)

[error] The imported variable isValidEmail is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)